galette/webroot/post_contribution_test.php

```diff
@@ -55,17 +58,31 @@
     $args = $_GET;
 }
 
-if (count($args) == 0) {
-    //we're called without arguments => exit.
-    die('No arguments.');
-}
-
 if (defined('STDIN')) {
-    //a successful script returns 0, we do not output anything
-    $fp = fopen(__DIR__ . '/cache/galette_post_contrib_file.txt', 'w');
+    if (empty($args)) {
+        //we're called without arguments => exit.
+        die('No arguments.');
```

Comment:
I guess this can be kept a first like it was, since `emtpy([])` will return true.

Reply:
`$args` is set from either `stream_get_contents`, `$_POST` or `$_GET`. In each case, if we receive no arguments, script must end.
Doing it globally makes things easier to understand; you changes do not impact that as far as I can see.